src/main.js

```diff
@@ -42,6 +42,8 @@ function deleteIssue(id) {
   }
 
   localStorage.setItem('issues', JSON.stringify(issues));
+
+    fetchIssues();
```

Comment:
@vopos, please lint your code! See CONTRINBUTING.md for details.

Reply:
@ehaberev, done!